lib/compress/zstd_compress.c

```diff
@@ -2568,13 +2574,19 @@ size_t ZSTD_initCStream_usingCDict(ZSTD_CStream* zcs, const ZSTD_CDict* cdict)
     return ZSTD_initCStream_usingCDict_advanced(zcs, cdict, fParams, 0);  /* note : will check that cdict != NULL */
 }
 
+/* ZSTD_initCStream_advanced() :
+ * pledgedSrcSize is optional: it can be 0, which means "unknown" if contentSizeFlag is set, or "unknown" if contentSizeFlag is not set. */
```

Comment:
Doesn't it mean zero if contentSizeFlag is set?

Reply:
Yes totally.
The comment has been changed in [`zstd.h`](https://github.com/facebook/zstd/pull/891/files#diff-71b1d36582455b38213ddb321e85c65bR737).
Maybe I should change it into `zstd_compress.c` as well.